modules/ROOT/pages/activity-tracking.adoc

```diff
@@ -1,87 +1,129 @@
-* The number of messages received in the file
-* Transmission flow +
-The *Transmission Flow* section of the summary information shows:
-* Success or failure of the transmission
-* Acknowledgment status +
-If the configured outbound message flow expects an acknowledgment from the trading partner or if the configured inbound message flow returns an acknowledgment to the trading partner, then you can view the acknowledgment (997 and TA1) payloads in the transmission detail.
+. Search for the transmission.
 +
-If you have not received an acknowledgment from your trading partner or if your trading partner sent a rejection acknowledgment, then you can review the acknowledgment status of outbound transactions.
-* Transformation and routing information
-+
-An exclamation point (!) appearing next to any of the rectangle boxes in the *Transmission Flow* section indicates an issue with the transmission.
-. Click *View Payload* to view and download the originally received payload.
+When you search for the transission, you can filter the results based on the following criteria:
+
+* Message type, such as `my-po-ack:ORD-9892330`
+* Date sent or received
 +
-You can examine the received payload to troubleshoot issues and find the issue location within the message flow.
+You can retrieve up to one week's worth of messages at a time.
 +
-* For outbound transmissions, you can view the source message payload received from your backend systems and the EDI message sent to your trading partner.
-* For inbound transmissions, you can view the received EDI message payload from your trading partner.
+* Acknowledgement status, such as `Overdue`
+* Messaging processing status, such as `Received`
+* Partner name
+* Partner message type, such as `X12 - 850`
+* Host message type, such as `JSON host-purchase-order`
+. Select the transmission to review summary information about the transmission and its message flow.
+
+== View Details for an Inbound Transmission
+
+The *Summary* section for an inbound transmission includes the following information:
+
+* Transmission status (a green checkmark for success or a red exclamation point for failure)
+* Transmission ID
+* Name of the transmitted file
+* Control number associated with the transmitted file, if available
+* Name of the receiving (Receive from Partners) endpoint
+* Date and time when the file was received
+* Message type, such as `mythical-po-ack-ORD-9892330`
+* Number of received messages in the file
+
+Take any of the following actions to view and download additional information about the inbound transmission:
+
+|===
+|Action |Steps
+
+|View and download the payload received from the partner (*Vijay, is this correct? How do users see the metadata for an inbound message?*)
+a|
+. In the *Transmission Flow* section, click *View Payload*.
+. Specify a wrap character, if needed.
+. Click *Download*.
+| View and download the MDN message sent to the partner
```

Comment:
View and download the MDN message sent to the partner for inbound AS2 transmissions.